### PR TITLE
OCPBUGS-26466: Disable PSA in non-previews 4.16

### PR DESCRIPTION
**Pull request by ibihim**
## What

Disable PSA in non-previews. Keep enabled in TechPreviewNoUpgrade and DevPreviewNoUpgrade

## Why

We didn't meet the threshold to enable it yet.

### CODE REVIEW

File: features/features.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ var (
 
 	FeatureGateOpenShiftPodSecurityAdmission = newFeatureGate("OpenShiftPodSecurityAdmission").
 							reportProblemsToJiraComponent("auth").
-							contactPerson("stlaz").
+							contactPerson("ibihim").
 							productScope(ocpSpecific).
 							enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```
Comment (deads2k) [inline features/features.go:78]: needs to be changed here and regenerated.

**Comment by ibihim** [comment]
PTAL @deads2k

**Comment by deads2k** [comment]
to pass verify, you must modify https://github.com/openshift/api/blob/master/features/features.go#L78 and regenerate.

**Pull request merged**